rest-backend/src/main/java/com/sap/psr/vulas/backend/model/Dependency.java

```diff
@@ -64,10 +64,7 @@
 @JsonInclude(JsonInclude.Include.ALWAYS)
 @JsonIgnoreProperties(ignoreUnknown=true, value={"traced", "reachableConstructIds", "touchPoints"}, allowGetters=true)
 @Entity
-@Table( name="AppDependency")
-///uniqueConstraints=@UniqueConstraint( columnNames = { "lib" , "app"} )
-// Note, the unique constraint is not defined as an annotation any longer as we need more expressiveness than what JPA allows
-// The new constraints are defined in the flyway migration V20180828.1730__depParent.sql as partial indexes
+@Table( name="AppDependency", uniqueConstraints=@UniqueConstraint( columnNames = { "lib" , "app", "parent", "relativePath"} ))
```

Comment:
@serenaponta : Do you think it is worthwhile to add a comment clarifying that the JPA-controlled unique constraint struggles with `null` values?

Reply:
Done